community/index/src/main/java/org/neo4j/index/internal/gbptree/GBPTree.java

```diff
@@ -202,6 +202,11 @@ public void startupState( boolean clean )
     static final Header.Reader NO_HEADER = (cursor,length) -> {};
```

Comment:
I think this should be renamed to `NO_HEADER_READER`. now that there's a `NO_HEADER_WRITER`